### PR TITLE
feat: removing some strategies

### LINKED ISSUE #3: Risk Framework: Remove Some Groups
Below groups can be deleted from risk framework.

- WBTC Badger levered
- League DAO staking
- Universe
- BArn DAO Staking

### PR DESCRIPTION
**Pull request by Majorfi**
Fixes #3

### CODE REVIEW

File: utils/risks.json [hunk 1]
```diff
@@ -641,7 +577,6 @@
 				"singlesided",
 				"ssc",
 				"Levered-AAVE-wBTC",
```
Comment (magic990619) [inline utils/risks.json:579]: This should be removed.

Comment (Majorfi) [inline utils/risks.json:579]:
```suggestion
				"ssc",
```

Comment (Majorfi) [reply utils/risks.json]: Missed this one, thank you

Comment (Majorfi) [reply utils/risks.json]: Fixed with [72d7bad](https://github.com/yearn/yearn-watch/pull/4/commits/72d7bad5d086015db2845615252f60a9cfa4ca3f)

**Comment by magic990619** [comment]
LGTM!

**Review by magic990619 — APPROVED** [review]

**Pull request merged**